### PR TITLE
update dashboard icons

### PR DESCRIPTION
**Pull request by jpoechill**
updates:
  – fix bug where 'Connected Profiles' was not highlighting correctly
  – add custom, and unique icons for each sidebar link
  – remove footer from sidebar for cleanliness and clarity (=> open for discussion)

Screen Shot 2019-07-31 at 7 38 43 PM
Screen Shot 2019-07-31 at 7 38 53 PM

### CODE REVIEW

File: assets/scss/base/djaodjin-dashboard.scss [hunk 1]
```diff
@@ -103,9 +103,10 @@ $dashboard-sidebar-color:    white;
 
 .dashboard-nav-main {
     float: left;
-    min-height: 100%;
-    padding-bottom: $dashboard-nav-footer-height; /* must be equal to .dashboard-nav-footer height
-                             in pixel */
+    min-height: 100vh;
```
Comment (smirolo) [inline assets/scss/base/djaodjin-dashboard.scss]: Apparently this change prevents the menu items in the sidebar to scroll when the sidebar content is longer than the browser window height.
Ex: cannot get to the "Rules" menuitem in the following use case:
Screen Shot 2019-08-01 at 9 27 51 AM

**Comment by jpoechill** [comment]
brought footer, and footer styles back. meant to keep footer changes separate from icons changes – hope to do so in a different PR.

**Comment by smirolo** [comment]
Sidebar seen as a provider (login as donny) looks good.
Sidebar seen as a subscriber (login as xia) has issues with menu item icons (see below).
Screen Shot 2019-08-18 at 2 30 53 AM

**Comment by smirolo** [comment]
There are 2 versions of the dash-icons committed to the PR (ex: user-wht.svg and user-blue.svg). This posses problems because the sidebar color and background are not always white and blue. 

The solution might be to follow the steps highlighted here [Multi-Colored SVG Symbol Icons with CSS Variables](https://frontstuff.io/multi-colored-svg-symbol-icons-with-css-variables) so that the color of the icons is inherited from `$dashboard-sidebar-color`  defined in `djaodjin-dashboard.scss`.

Two versions of the icons for when it is displayed on a light or dark background might still be useful.

**Pull request closed without merging**